modules/ipi-install-network-requirements.adoc

```diff
@@ -119,6 +119,12 @@ Your DHCP server must provide a DHCP expiration time of 4294967295 seconds to pr
 External load balancing services and the control plane nodes must run on the same L2 network, and on the same VLAN when using VLANs to route traffic between the load balancing services and the control plane nodes.
 ====
 
+[IMPORTANT]
+.Do not change IP addresses manually after deployment
```

Comment:
Two nits:

Should unschedulable be in backticks since it's a variable?

There's no link or additional section for "Working with nodes." Not sure where users will be directed here.

Reply:
It's under "Nodes", but I cannot put in a hyperlink into a module at this time. That's still the case, correct? I didn't put unschedulable into backticks, because I got the information from other docs that also do not use back ticks, so I was trying to be consistent there.

Reply:
You are correct, @johnwilkins. No xrefs allowed in modules, so referring to the section in quotes is our current practice.

I do agree with @stevsmit that `unschedulable` should be in backticks per our docs guidelines for variables and other non-GUI items: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference

But I wouldn't mark it as incredibly urgent considering present timing.